### ISSUE TITLE
Use a different system for "calling on" people in the team meeting

### ISSUE THREAD

**Issue by choldgraf** [jupyterhub/team-compass#157]
I've noticed that our "add your name to the speaking queue" isn't working...we don't tend to use it and people just speak when they wish. I'm still worried that this is creating an environment where it's hard for some members of the team calls to speak. I was speaking with Tracy Teal (from the Carpentries) the other day, and she mentioned that they use a simpler system"

for each meeting, one person's job is to "call" on people to speak. People type the word "hand" into the chat room, and the person just calls on that person's name in the order that they typed.

I think this might be easier to actually adopt because it doesn't involved scrolling back and forth in the notes. The only extra downside is the added role of calling on people.

Anybody interested in trying this out for the next meeting?

**Comment by betatim** [comment]
I like the suggestion. Do you know if we can somehow turn off the zoom chat notification sound? Otherwise we might go mad with "ding ding ding ... ding ding ...." noises.

Another thing we should work on is enforcing the rules which should be the job of the person doing the calling on but it would be helpful if the rest of the team could help reinforce it.

Related but somewhat off topic: for the short updates/shoutouts we should keep working on keeping them "no questions, no feedback" so they don't become agenda items through the backdoor.

**Comment by choldgraf** [comment]
Good question - I'm not too sure about the sounds, I actually haven't noticed them before.

If we'd like to try this, should we give the role to somebody else, or should this be part of the meeting moderator's role? I'd be curious to hear what @Zsailer or @lheagy think about it, since they have recent experience moderating meetings as well :-)

**Comment by Zsailer** [comment]
I agree, the speaking queue thing doesn't really work on video calls. I already stopped using it in the Jupyter Community Call. I'm fine trying a new approach like using the chat room. I think the moderator can handle this. 

In my experience, being the call moderator basically takes you out of the conversations. I find myself more focused on making sure the call runs smoothly, tracking time, remembering to record, and transitioning topics. This isn't a *ton* of responsibility, but it does distract you. Adding this speaking queue is just a small addition to the list of responsibilities. Giving it to another person would likely distract them from the conversation too, thus taking two people out of the call. I think we should just keep the call mechanics under a single role.

**Comment by Zsailer** [comment]
As a side comment...
The challenge with implementing a "call on speaker" mechanism is that it's not always friendly to newcomers. Unless we actively explain it at the beginning of every meeting, newcomers may "break the rules" without knowing it. This can be frustrating for others on the call who might feel cheated, and awkward for the moderator who is expected to correct the newcomer. However, correcting them can discourage newcomers from participating. I would hate to scare people away because of our process. 

On the other hand, a speaker queue also prevents any one person from _dominating_ the call. This can actually promote newcomers to participate. I'm not sure I know which outcome is more likely. 

What do you all think? Is a speaking queue actually helpful or distracting?

**Comment by betatim** [comment]
I think what ever system we use, it needs explaining at the start of every meeting. We should remember to do that and maybe word smith a short, efficient and good explanation that we can read out each time.

I think having a system that all the regulars support and use (so it isn't just the moderator who has to do it and be the bad cop) helps with not scaring away people (if it is explained at the start). IMHO as long as it is fair (which means strict enforcement) I don't see who it would scare away. I can see it would annoy those who usually dominate or take advantage of unstructured meetings but that is the system working as designed.

Maybe splitting the job of moderating the queue and keeping us on track/moving along in the agenda are two jobs so we should have to people.

**Comment by choldgraf** [comment]
Another option is to include a short "meeting format" section at the bottom of the team-compass issue, as well in the Discourse post. We can briefly mention this format at the start of each meeting to set the tone. Though as @betatim mentions, I think the most important thing will be core members following the rules.

to @Zsailer's point about moderating sort-of taking you out of the conversation, I'd love if we created more processes that made it easier to rotate the meeting leader so that we aren't putting undue burden on one person.

**Comment by lheagy** [comment]
Agreed, the speaking que wasn’t used at all in the last meeting. I hesitate to put it in the chat, as right now, people are using it to add a thought, comment, joke or some other subtext in the meeting. If we try to have it be dual-purpose: current use + speaking que, I think we will lose some of the current chat interactions. I think there is a “raise hand” functionality in Zoom (though I have no idea what it does...): has anyone ever tried it? 

+1 on including a short: “meeting format” description in the discourse post.

**Comment by choldgraf** [comment]
I haven't used any in-built functionality for that before, sounds like it's possible from here:

https://support.zoom.us/hc/en-us/articles/115001286183-Nonverbal-Feedback-During-Meetings

I'm happy w/ whatever non-verbal solution...just suggested the "hand" chat method because I think that's what the Carpentries use, and I've found them to be a pretty well-organized distributed community

**Comment by betatim** [comment]
Sounds like we have enough consensus to try a new iteration of the speaking queue.

I think the system we should trial next is:
1. use the zoom "raise hand" button to signal you want to speak
2. have one person who runs the meeting and one person who deals with the queue
3. short statement that explains it all in the meeting notes, agenda, discourse. IMHO we should take 30s to read it out at the start of the meeting for the first few times till it becomes more routine.

I propose the following text for (3):

For discussions we use a "speaker queue" system. This means that if you'd like to say something or ask a question you should raise your hand (use the "hand" button in Zoom) to get yourself added to the queue. When it is your turn you will be called on by the moderator of the queue. If by the time you are called on your point has already been made or isn't relevant any more you can pass.

**Comment by minrk** [comment]
Sounds like a good system. I don't think I got the hang of the current speaker queue or understood what rules we had before, so a clear explanation of the system at each call (both read and written) would be great.

**Comment by choldgraf** [comment]
I like @betatim's proposal - would anybody like to demo this on the calpoly zoom so we can see what actually happens when somebody clicks "raise hand"? I'll be around the rest of the day! Maybe @Zsailer and I can find a time and then we can document a screenshot or something

**Comment by Zsailer** [comment]
Yeah, I'd be happy to help. I'll ping you on the JupyterHub gitter.
